### DISCUSSION TITLE
How to make "Open Link in New Window" opens the PWA correspond to the link?

### DISCUSSION THREAD

**Discussion by KSXGitHub** [filips123/PWAsForFirefox#416] [Ideas]
For example, I have both GitHub and YouTube installed as PWAs with the Default profile. On a README.md, there is a YouTube link, so I expect right-click → Open Link in New Window to open the YouTube PWA (just like how it works in Chrome). But currently, doing so would open a GitHub PWA windows with address bar and content from YouTube.

**Comment by filips123** [comment]
I think this is currently not possible, but I may implement it in the future.
